src/test/java/com/artipie/rpm/CrCompareDependencyTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/rpm-adapter/LICENSE.txt
+ */
+package com.artipie.rpm;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit tests for {@link CrCompareDependency}.
+ *
+ * @since 1.9.9
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+final class CrCompareDependencyTest {
```

Comment:
I'd suggest to add several more tests, here are example data sets:
```
                                 <rpm:entry name="libc.so.6"/>
				<rpm:entry name="libc.so.6(GLIBC_2.0)"/>
				<rpm:entry name="libc.so.6(GLIBC_2.1)"/>
				<rpm:entry name="libc.so.6(GLIBC_2.1.1)"/>
				<rpm:entry name="libc.so.6(GLIBC_2.1.3)"/>
				<rpm:entry name="libc.so.6(GLIBC_2.2)"/>
				<rpm:entry name="libc.so.6(GLIBC_2.3)"/>
				<rpm:entry name="libc.so.6(GLIBC_2.3.4)"/>
				<rpm:entry name="libc.so.6(GLIBC_2.4)"/>
```
and also cover the case when same items are compared.

Reply:
Done.